source/install/linuxone.html.md

```diff
@@ -0,0 +1,221 @@
+
+## Software
+Red Hat Enterprise Linux (RHEL) 8 is the only verified version at the creation time of this how-to, we will update this guide if we verify other releases and Linux distributions. 
+
+## Hardware 
+Here is the minimum recommended configuration for an LPAR of IBM Z or IBM LinuxONE system. It should have at least 16GB RAM, 2 logical IFLs configured with hardware virtualization extensions, and two network adapters.
+
+| Server | CPU | Memory | Storage | Network Adapter |
+| ------ | --- | ------ | ------- | --------------- |
+| Physical Servers| 2 logical IFLs | >= 16GB | 50 GB | 1 x Admin NIC (can be shared OSA port) 1 x Data NIC (dedicate OSA port) | 
+
+*If you perform a single-node deployment, please allocate more memory on the host to gain a stable deployment. 
+
+## Install Operating System on an LPAR
+
+Install a base operating system on the LPAR, minimum installation is OK.
+ 
+Follow below RHEL8 official installation guide, if you are not familiar with the installation process. 
+https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_installation/installing_red_hat_enterprise_linux_on_ibm_z
+
+## Disable NetworkManager
+
+For a working and stable deployment, below services are recommended to be disabled and stopped, but with the traditional network service enabled and up.
+
+Follow this guide for the details:
+* https://access.redhat.com/solutions/4365931 
+
+Issue below commands on the terminal of every participated node.
+
+```Shell
+systemctl disable firewalld.service
+systemctl stop firewalld.service
+systemctl disable NetworkManager.service
+systemctl stop NetworkManager.service
+systemctl enable network.service
+systemctl start network.service
+```
+
+## Make sure your IP and hostname were listed in /etc/hosts.
+```Shell
+cat /etc/hosts
+
+172.16.36.194   	lpar94
+```
+# Software repositories
+## Enable YUM repos
+You need enable Redhat subscription to get update packages from Red Hat, and add following repos to get OpenStack packages and OpenStack dependency packages.
+
+```Shell
+[rdo-train-upstream]
+name=rdo-train-upstream
+baseurl=https://trunk.rdoproject.org/rhel8-train/current-tripleo/
+enabled=1
+gpgcheck=0
+[rdo-train-linuxone-deps]
+name=rdo-train-linuxone-deps
+baseurl=http://linuxone.cloud.marist.edu:8080/repos/rdo/rhel8.0/deps/
+enabled=1
+gpgcheck=0
+```
+
+## Make sure to use latest yum repo
+```Shell
+yum clean all
+```
+
+## Disable virt module by issuing following command
+```Shell
+dnf module disable virt
+```
+
+#  Installing Packstack Installer
+
+## Prepare the answer file
+
+We use Openvswitch for network support, make sure your packstack answer file has following lines:
+
+```Shell
+CONFIG_NEUTRON_ML2_MECHANISM_DRIVERS=openvswitch,l2population
+CONFIG_NEUTRON_L2_AGENT=openvswitch
```

Comment:
Some reason to not use ovn instead of openvswitch?, ovn is default in train

Reply:
Thanks @amoralej. 
So far, OVN is not supported on LinuxONE to work with OpenStack. We might can use OVN in the future. Now OpenVswitch is recommended for OpenStack network on LinuxONE.